src/blueapi/utils/base_model.py

```diff
@@ -0,0 +1,35 @@
+from pydantic import BaseConfig, BaseModel, Extra
+
+
+def _to_camel(string: str) -> str:
```

Comment:
Some words on why this change is required would be useful to prevent it being undone again at some latter point

Reply:
Done in docstring of `BlueapiPlanModelConfig`